src/fusor/translator.py

```diff
@@ -84,6 +94,10 @@ def _format_fusion(
             params["readingFramePreserved"] = rf
         if assay:
             params["assay"] = assay
+        if contig:
+            params["contig"] = contig
+        if reads:
+            params["readData"] = reads
```

Comment:
Conditional isn't necessary (missed this when reviewing above lines too) since the default for both function signature and pydantic model are null

```suggestion
        params["contig"] = contig
        params["readData"] = reads
```

Reply:
Consider changing first line to 
```python
params = {
  "causativeEvent": ce,
  "readingFramePreserved": rf,
  "assay": assay,
  "contig": contig,
  "readData": reads
}
```